### PR TITLE
got message from server

### PR DESCRIPTION
**Pull request by seunomonije**
From the walkthrough

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jessicaslaughter) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:45]: nit: remove blank line

File: portfolio/src/main/webapp/script.js
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/script.js]: nit: new line at EOF

File: portfolio/src/main/webapp/script.js
Comment (jessicaslaughter) [inline portfolio/src/main/webapp/script.js]: nit: there should be a space before the curly brace

**Comment by jessicaslaughter** [comment]
Did you mean to make this PR a merge into the master branch instead of the seun2 branch or are you going to do a larger merge from seun2 into master later?

**Review by jessicaslaughter — CHANGES_REQUESTED** [review]

**Comment by seunomonije** [comment]
Was gonna do all the server-side on this branch little by little while waiting for seun2 to be ready, then once seun2 was ready, merge server-side into seun2, then seun2 into master

**Comment by jessicaslaughter** [comment]
Even though you're doing all server-side work on this branch, you should still create a separate PR every time you want us to review new work. Once a PR has been sent out for review, you shouldn't add any new functionality to it since we're actively reviewing the original code. So instead of committing new work to a PR that's already being reviewed, you should create a new one and add us as reviewers once it's in a state that you want it to be reviewed.

**Comment by jessicaslaughter** [comment]
If you want us to wait on reviewing this one until you think it's got everything you want in it (while keeping it small), you can just remove us from the reviewers list and we'll wait!

**Comment by seunomonije** [comment]
Ah ok I see thank you

**Comment by seunomonije** [comment]
So I'm gonna have to have a bunch of branches and then delete them once I'm done right? It also makes sense with rebasing now

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jpnickolas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Private functions should get at the bottom of the file after all public files.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jpnickolas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Only add one space after the semicolon but before the comment.

File: portfolio/src/main/webapp/script.js
Comment (jpnickolas) [inline portfolio/src/main/webapp/script.js]: remove debug statement

**Review by jpnickolas — CHANGES_REQUESTED** [review]

**Comment by jessicaslaughter** [comment]
Yup! Ideally you'd have a branch 1 that points to master with some changes, and another branch 2 that points to branch 1 with different changes, and each branch eventually has a PR with all its commits. Once the branch 1 PR is reviewed and accepted, you push it to master, fix any merge conflicts in branch 2, and rebase that branch 2 to point to master.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jessicaslaughter) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: The asterisks should be aligned vertically, and the second line of the comment should line up with the top line on the left (there shouldn't be extra white space between the asterisk and that text)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jessicaslaughter) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: return statement should be indented

**Review by jessicaslaughter — CHANGES_REQUESTED** [review]

**Review by jessicaslaughter — APPROVED** [review]

**Review by jpnickolas — APPROVED** [review]

**Pull request merged**
